spec/beaker-pe/pe-client-tools/executable_helper_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require 'beaker'
+require 'scooter'
```

Comment:
@tvpartytonight Yes I think we should wait for that, and it's good to hear that Scooter is going public, since that simplifies using it more broadly in beaker-pe.  I only ran into because I was running the specs and didn't have GEM_SOURCE declared locally.